Dockerfile

```diff
@@ -2,11 +2,17 @@
 FROM python:3.9
 
 # Set working directory.
+WORKDIR /code
 
 # Expose port 8000 for your FastAPI server.
+EXPOSE 8000
 
 # Install Python requirements using Pip.
+COPY requirements.txt ./code/
+RUN pip install -r /code/requirements.txt
 
 # Copy your codebase into the Docker container.
+COPY ./ ./code
 
 # Run FastAPI server on the port exposed above.
+CMD ["uvicorn", "main.app"]
```

Comment:
You need to specify the host address here ([reason](https://stackoverflow.com/a/20778887)).